modules/serverless-admin-broker-default.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+//  * serverless/admin_guide/serverless-configuring-eventing-defaults.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-broker-default_{context}"]
+= Configuring the default broker implementation
+
+You can specify the default broker implementation for the cluster, or for one or more namespaces.
+
+To do this, you must configure the `defaultBrokerClass` spec in the `KnativeEventing` custom resource (CR), as well as adding configuration for the `config-br-defaults` config map.
+
+Currently, the available broker class options for {ServerlessProductName} are:
+
+* The `MTChannelBasedBroker` class, which uses the default Knative multi-tenant, channel-based broker.
+
+* The `Kafka` broker class, which uses the Knative Kafka broker (currently Technology Preview).
```

Comment:
Just setting there `Kafka` does not directly make it work. We need to also loop in some general configuration for the kafka bit (e.g. what is the URL of the bootstrap server etc).

Usually this is (globally) captured on the `kafka-broker-config` ConfigMap (that basically contains the configuration from the `KnativeKafka.Spec.Broker` configuration option.

**NOTE:** while for the default `MTChannelBasedBroker` it all seems to work by just setting the name, if you look at the details there is also additional config needed. In that MT-Channel broker case it does point to the `config-br-default-channel` config map.  So, it's very similar - but still a bit magic, b/c those settings are defaults, hence they work better out of the box.


But for kafka, I'd change the text somehow to say "you need to point to the configmap tat captures your configuration of `KnativeKafka`, which is the `kafka-broker-config` CM in the `knative-eventing` system namespace

Reply:
I mentioned the configmaps in the callouts and showed examples of the defaults already:
https://github.com/openshift/openshift-docs/pull/42899/files#diff-2077b3964638b30b796d7d4c83ed4f14f446acbf87205d2d084bcbed288c6fffR66-R107

Is this not correct?

Reply:
No, that is perfect and correct - I just wanted to make sure, that just (and only) adding `Kafka`, does not help - because for `Kafka` we need to reference different config maps as well.

That's all - Just clarifications :sweat_smile: